src/components/Alerts/AlertsWrapper.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { ReactNode } from "react"
+import { v4 as uuid } from "uuid"
+import { placementType, AlertProps } from "./AlertInterface"
+import TransitionWrapper from "./TransitionWrapper"
+
+export interface AlertsWrapperProps {
+  // Pass value of where placement where alert should appear
+  placement?: placementType
+  // Container define which component encapsulate as box to alerts,
+  getContainer?: () => HTMLElement | ReactNode
+}
+
+export interface AlertsWrapperStates {
```

Comment:
This should instead be called `AlertStatesWrapper`
`AlertWrapperStates` would imply that it contains the states for the alertWrapper. However I beleive its instead a wrapper that contains the states of alerts.